tooltip.browser.json

```diff
@@ -0,0 +1,8 @@
+{
+    "dependencies": [
+        {
+            "path": "./dist/tooltip/ds6/tooltip.css",
+            "if-flag": "skin-ds6"
```

Comment:
We should consider how we deal with this in DS4. It's under Notice in DS4, and we might want to break out the DS4 tooltip from the DS4 notice so we can do the dependency for DS4 when folks are using the eBayUI component.

Reply:
See my requested changes comment.